site/docs/hosting/vps.md

```diff
@@ -2,24 +2,124 @@
+You probably don't need it, so you can start by renting a server for a month, which is much cheaper.
+
+In any case, Hostinger provides a 30-day money-back guarantee.
+:::
+
+After payment you will be able to set up your server:
+
+1. **Location.**
+   We recommend that you choose the location closest to Amsterdam.
+   It is in Amsterdam that the Telegram server is located, which is responsible for ensuring the operation of bots.
+   If you use your own Bot API server, choose the location closest to it.
+2. **Server type.**
+   Choose the option "Clean OS".
+3. **Operating system.**
+   We will use Ubuntu 22.04.
+   If you choose a different system, some steps may be different, so be careful.
+4. **Server name.**
+5. **Root password.**
+   Make a strong password and keep it in a safe place!
+6. **SSH key**.
+   Skip this step.
+   We will set up SSH keys [later](#ssh-keys).
+
+After the server is created, you can connect to it using SSH:
+
+> SSH (_Secure Shell_) is a network protocol for remote control of a computer (server) and file transfer.
 
-## systemd
+```sh
+ssh root@<ip-address>
+```
+
+replacing `<ip-address>` with the IP address of your server, which you can find on the server management page.
+
+::: tip Separate User for Each Application
+In this guide, all actions with the server will be performed as the root user.
+This is done on purpose to simplify the material.
+However, in reality, the root user should only be responsible for general services (web server, database, etc.), and applications should be run by individual non-root users.
+This approach ensures the security of confidential data and prevents hacking of the entire system.
+At the same time, it imposes some inconveniences.
+Describing all these points unnecessarily increases the complexity of the article, which we try to avoid.
+:::
+
+## Starting the Bot
+
+We now have a server at our disposal where we can run the bot to keep it running around the clock.
+
+> To simplify the beginning of the article, we skipped the step of delivering the code to the server automatically each time after pushing your code, but it is described [below](#ci-cd).
+>
+> For now, you can copy local files to a remote server using the following command:
+>
+> ```sh
+> scp <path-to-local-files> root@<ip-address>:<path-to-remote-directory>
+> ```
+>
+> replacing `<path-to-local-files>` with the path to local files, `<ip-address>` with the IP address of your server, and `<path-to-remote-directory>` with the path to the directory where the bot's source code should be stored on the server.
+
+:::tip Don't forget to install the runtime!
+To run the bot, you need to install Node.js or Deno on the server, depending on the runtime in which the bot will run.
+This is beyond the scope of this article, so you will need to do it yourself.
+You probably already did this when [getting started](../guide/getting-started.md), so you should be familiar with the steps. :wink:
```

Comment:
This emoji does not render properly

Reply:
What do you mean by that? It will be displayed for me

Reply:
Can you post a screenshot?

Reply:
![Screenshot from 2023-07-11 07-50-33](https://github.com/grammyjs/website/assets/75927667/a4511c6d-4c39-478b-8589-fdb20992ffdb)

Reply:
![image](https://github.com/grammyjs/website/assets/74030149/9dabcf8c-644c-47d8-98f5-1eaf9131c716)

Reply:
Right, it's using the system fonts. This will surely be fixed by Vitepress at some point.